### PR TITLE
Allows multiple creators

### PR DESCRIPTION
**Pull request by gsimone**
This allows multiple creators by just using an array - with objects of the same shape as the regular creator. 

eg.
```json
{
  "name": "Bunny",
  "creator": [{
    "name": "McGuire Computer Graphics Archive",
    "link": "https://casual-effects.com/g3d/data10/index.html#"
  }, {
    "name": "McGuire Computer Graphics Archive",
    "link": "https://casual-effects.com/g3d/data10/index.html#"
  }],
  "license": 1,
  "category": "animals"
}

```

image

### CODE REVIEW

**Comment by SaraVieira** [comment]
Oh, you forgot to comment the API thing back

**Comment by SaraVieira** [comment]
Also can you put a screenshot pls?

File: src/components/ModelInfo.js [hunk 1]
```diff
@@ -11,6 +11,17 @@ function classNames(...classes) {
   return classes.filter(Boolean).join(' ')
 }
 
+function Creator(props) {
+
+  console.log("creator ", props)
```
Comment (SaraVieira) [inline src/components/ModelInfo.js:16]: *shakes head in disapproval*

Comment (gsimone) [reply src/components/ModelInfo.js:16]: oh shi

**Comment by gsimone** [comment]
Should I move the Creator to a dedicated file?

**Comment by SaraVieira** [comment]
> Should I move the Creator to a dedicated file?

Yes, since there is also a material info

**Pull request merged**